$OpenBSD$

try the PIC_INITIAL_EXEC which seems to primarily serve to make things faster at the cost
of not being able to link the monolib later ??

Index: mono/utils/mono-tls.c
--- mono/utils/mono-tls.c.orig
+++ mono/utils/mono-tls.c
@@ -54,7 +54,7 @@
  * Define this if you want a faster libmono, which cannot be loaded dynamically as a
  * module.
  */
-//#define PIC_INITIAL_EXEC
+#define PIC_INITIAL_EXEC
 
 #if defined(PIC)
 
